pandas/core/generic.py

```diff
@@ -2015,16 +2015,19 @@ def sort_values(self, by, axis=0, ascending=True, inplace=False,
             Sort ascending vs. descending
         inplace : bool, default False
             if True, perform operation in-place
+        sort_remaining : bool, default True
+            if true and sorting by level and index is multilevel, sort by other
+            levels too (in order) after sorting by specified level
+
+            .. versionadded:: 0.19.0
```

Comment:
Ah, you wanted to add it for kind and na_position I suppose. The `versionadded` applies to the keyword before it, so should be placed after each keyword that is added. 

Only, the problem here is that those keywords are not added in 0.19.0 for DataFrame, but only for Series. And given that the docstring is reused, this makes this a bit difficult. I would just leave it out in this case then.